docs/bigquery/snippets.py

```diff
@@ -89,476 +97,556 @@ def do_something_with(_):
-    dataset.create()
+    client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    table = dataset.table(TABLE_NAME, SCHEMA)
-    table.friendly_name = ORIGINAL_FRIENDLY_NAME
+    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
     table.description = ORIGINAL_DESCRIPTION
-    table.create()
+    table = client.create_table(table)
     to_delete.insert(0, table)
 
-    # [START table_patch]
-    assert table.friendly_name == ORIGINAL_FRIENDLY_NAME
+    # [START update_table_simple]
     assert table.description == ORIGINAL_DESCRIPTION
-    table.patch(
-        friendly_name=PATCHED_FRIENDLY_NAME,
-        description=PATCHED_DESCRIPTION,
-    )      # API request
-    assert table.friendly_name == PATCHED_FRIENDLY_NAME
-    assert table.description == PATCHED_DESCRIPTION
-    # [END table_patch]
+    table.description = UPDATED_DESCRIPTION
 
+    client.update_table(table, ['description'])  # API request
+
+    assert table.description == UPDATED_DESCRIPTION
+    # [END update_table_simple]
 
-def test_table_update(client, to_delete):
+
+def test_update_table_multiple_properties(client, to_delete):
     """Update a table's metadata."""
-    DATASET_NAME = 'table_update_dataset_%d' % (_millis(),)
-    TABLE_NAME = 'table_update_table_%d' % (_millis(),)
-    dataset = client.dataset(DATASET_NAME)
+    DATASET_ID = 'update_table_multiple_properties_dataset_%d' % (_millis(),)
+    TABLE_ID = 'update_table_multiple_properties_table_%d' % (_millis(),)
+    dataset = bigquery.Dataset(client.dataset(DATASET_ID))
     dataset.description = ORIGINAL_DESCRIPTION
-    dataset.create()
+    client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    table = dataset.table(TABLE_NAME, SCHEMA)
+    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
     table.friendly_name = ORIGINAL_FRIENDLY_NAME
     table.description = ORIGINAL_DESCRIPTION
-    table.create()
+    table = client.create_table(table)
     to_delete.insert(0, table)
 
-    # [START table_update]
+    # [START update_table_multiple_properties]
     assert table.friendly_name == ORIGINAL_FRIENDLY_NAME
     assert table.description == ORIGINAL_DESCRIPTION
-    NEW_SCHEMA = table.schema[:]
-    NEW_SCHEMA.append(SchemaField('phone', 'string'))
+
+    NEW_SCHEMA = list(table.schema)
+    NEW_SCHEMA.append(bigquery.SchemaField('phone', 'STRING'))
     table.friendly_name = UPDATED_FRIENDLY_NAME
     table.description = UPDATED_DESCRIPTION
     table.schema = NEW_SCHEMA
-    table.update()              # API request
+    table = client.update_table(
+        table,
+        ['schema', 'friendly_name', 'description']
+    )  # API request
+
     assert table.friendly_name == UPDATED_FRIENDLY_NAME
     assert table.description == UPDATED_DESCRIPTION
     assert table.schema == NEW_SCHEMA
-    # [END table_update]
+    # [END update_table_multiple_properties]
 
 
-def _warm_up_inserted_table_data(table):
+def _warm_up_inserted_table_data(client, table):
```

Comment:
I believe this function is no longer used by the snippets and can be deleted.

Reply:
Removed in e30c19a